### PR TITLE
Feature/us1258 stripe ach customer payment

### PR DESCRIPTION
**Pull request by IngageRitterS**
Pull request ready for review.  NOTE - the code needs to be DRY'd up.  However, when the code is refactored to remove some duplication, the chaining of the promises is not functioning properly.  Any and all comments/suggestions on how to alleviate this issue are welcomed!

### CODE REVIEW

File: crossroads.net/app/give/bankCreditCardDetails.html [hunk 1]
```diff
@@ -10,8 +10,8 @@
         </a>
     </span>
 
-    <section class="bank-section" ng-if="give.dto.view=='bank'">
-    	<bank-info></bank-info>
+    <section ng-if="give.dto.view=='bank'">
+    	<bank-info ng-init="give.dto" account="give.dto.account" bankinfo-submitted="give.bankinfoSubmitted" routing="give.dto.routing"></bank-info>
```
Comment (silbermm) [inline crossroads.net/app/give/bankCreditCardDetails.html]: what does `ng-init="give.dto"` do?

File: crossroads.net/app/give/give_controller.js [hunk 2]
```diff
@@ -49,6 +49,9 @@
         vm.processing = false;
```
Comment (silbermm) [inline crossroads.net/app/give/give_controller.js]: according to https://github.com/crdschurch/angular-styleguide#style-y033 , methods within the controller should be alphabetized

File: crossroads.net/app/give/services/payment_service.js [hunk 3]
```diff
@@ -10,6 +10,7 @@
       donor : getDonor,
```
Comment (silbermm) [inline crossroads.net/app/give/services/payment_service.js]: according to https://github.com/crdschurch/angular-styleguide#style-y033 , methods should be alphabetized

**Comment by IngageRitterS** [comment]
Alphabetization corrected.  ng-init removed.

**Comment by silbermm** [comment]
:tada:

**Pull request merged**